test/sequential/sequential.status

```diff
@@ -40,6 +40,10 @@ test-inspector-contexts : SKIP
 test-inspector-scriptparsed-context : SKIP
 test-inspector-stop-profile-after-done : SKIP
 
+[$jsEngine==chakracore && $system==win32]
+# These tests are failing for Node-Chakracore and should eventually be fixed
```

Comment:
I don't think this comment is accurate for this case.  We should probably just say that this test can fail depending on the ports that Windows services/processes are using.

Reply:
Good point, thanks.